index.js

```diff
@@ -277,9 +282,45 @@ app.post('/notify', function(req, res) {
       throw new Error('Not Found');
     }
 
+    // check activity and sendNotification if not specified or "1"
+    function checkActivity(token, machine, client, registration, payload, ttl) {
+      // XXX: this should be a hash per machine
+      var machineKey = token + ':' + machine;
```

Comment:
It is - comment should be removed, also I think  the key should be {machine}_clients